README.md

```diff
@@ -3,13 +3,22 @@ This is application is a simple tic tac toe game that is created utilizing ruby
 
 ## Starting the Game
```

Comment:
I believe you are missing an important setup step. When using gems we need to run "bundle install"

Reply:
added this instruction to the readme